webrender/src/clip_scroll_node.rs

```diff
@@ -452,6 +434,39 @@ impl ClipScrollNode {
                     info.current_offset + state.parent_accumulated_scroll_offset;
             }
         }
+
+        // Store coord system ID, and also the ID used for shaders to reference this node.
+        self.coordinate_system_id = state.current_coordinate_system_id;
+        self.id = ClipScrollNodeIndex(node_data.len() as u32);
```

Comment:
might be useful to have the name of this type reflecting the fact it's for GPU/shader consumption only